metadata-db-app/src/migrations/050_setup_variable_associations_table.clj

```diff
@@ -0,0 +1,60 @@
+(ns migrations.050-setup-variable-associations-table
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private variable-assocs-column-sql
+  "id NUMBER,
+  concept_id VARCHAR(255) NOT NULL,
+  native_id VARCHAR(1500) NOT NULL,
+  metadata BLOB NOT NULL,
```

Comment:
@indiejames, We don't have NULL value as metadata for tombstones in production, it is always a 20 bytes BLOB which is the encrypted empty string. The production schema does allow NULL, but that is not from the db migration code. It must have been done manually as part of the migration to AWS. I agree that we should probably handle tombstone metadata more efficiently, but that is outside of the scope of this issue. I will leave it to @chris-durbin if he wants to file a separate issue or not.